src/policy/immix/immixspace.rs

```diff
@@ -77,6 +78,15 @@ impl<VM: VMBinding> SFT for ImmixSpace<VM> {
         #[cfg(feature = "global_alloc_bit")]
         crate::util::alloc_bit::set_alloc_bit(_object);
     }
+    #[inline(always)]
+    fn sft_trace_object(
+        &self,
+        _trace: SFTProcessEdgesMutRef,
+        _object: ObjectReference,
+        _worker: GCWorkerMutRef,
+    ) -> ObjectReference {
+        panic!("We do not use SFT to trace objects for Immix. sft_trace_object() cannot be used.")
```

Comment:
Is this just for performance reasons?

Reply:
Yes. I got some strange results for immix. I benchmarked 3 builds: master, immix with SFTProcessEdges in the PR, immix with the original ImmixProcessEdges in the PR. Somehow, the last two showed a similar slowdown, and I couldn't figure out the reason (I expect immix with the original ImmixProcessEdges should behave the same as master). So in the end, I remove all the changes related with immix in the PR.

Maybe I should dig more into the performance for immix.

Reply:
I did another measurement for this PR only using `pmd` with `immix` (which I found usually has the worst slowdown). I had 200 invocations so I assume the result is robust enough. There is slowdown of 0.7% total time and 2.8% STW time. There is no difference in retired instructions, but the CPU cycles increases by 3.6% (for GC). There are also some difference in some major GC work packets as well. 

![pmd-immix](https://user-images.githubusercontent.com/1664709/153523053-e363b2d5-9bc9-4618-97f0-1cc37455de70.png)

I couldn't see how this PR affects immix performance.

Reply:
This is genuinely bizarre. Are there more cache misses perchance?

Reply:
> This is genuinely bizarre. Are there more cache misses perchance?

I didn't measure that, but I could add that the measurement.